support-frontend/app/controllers/Application.scala

```diff
@@ -234,6 +239,45 @@ class Application(
     )()).withSettingsSurrogateKey
   }
 
+  def contributionsCheckoutInEpic(countryCode: String): Action[AnyContent] = PrivateAction { implicit request =>
+    implicit val settings: AllSettings = settingsProvider.getAllSettings()
+
+    val geoData = request.geoData
+    val serversideTests = generateParticipations(Nil)
+    val uatMode = testUsers.isTestUser(request)
+    val guestAccountCreationToken = request.flash.get("guestAccountCreationToken")
+
+    Ok(
+    views.html.contributions(
```

Comment:
Currently we're just reusing the contribs template, potentially we could create a custom one. It's not straightforward to see exactly what is/isn't required here though

Reply:
Having thought about this a little more I think it's probably simplest & easiest to just reuse the existing template